content/How to customize fonts in React Native - Manuela Viola.md

```diff
@@ -0,0 +1,109 @@
+# How to customize fonts in React Native
+
+![Main image](images/customize_fonts_main.png)
+
+## Introduction
+
+Like every newbie in Rootstrap, my first assignment was to develop Target MVD application, in my case, in React Native. While doing this, one of the first things I tried was to set the family font that the application uses, and I found out it wasn't as easy as I expected.
```

Comment:
Reader will not know what's a Target application, I'll rephrase to something like:
```My first assignment was to develop a React Native application, that involved maps, chat and following the design specifications to the letter. ```